app/controllers/friendships_controller.rb

```diff
@@ -1,13 +1,13 @@
 class FriendshipsController < ApplicationController
   def create
-    BackEndFacade.add_friend(create_friendship_params)
+    FriendshipFacade.add_friend(create_friendship_params)
     flash[:alert] = "You've just added a Swolemate! Now go get them gains!!"
-    redirect_to dashboard_path(current_user.id)
+    redirect_to dashboard_index_path
   end
 
   def destroy
-    BackEndFacade.delete_friend(destroy_friendship_params)
-    redirect_to dashboard_path(current_user.id)
+    FriendshipFacade.delete_friend(destroy_friendship_params)
+    redirect_to dashboard_index_path
     flash[:alert] = 'Swolemate removed!'
```

Comment:
Assuming folks agree with nomenclature, we should update to `SwoleMate`.  Not picky on what we choose, just looking for consistency.

Reply:
> Assuming folks agree with nomenclature, we should update to `SwoleMate`. Not picky on what we choose, just looking for consistency.

@tvaroglu Oh! I like that! The dilemma is figuring out how to differentiate `friends` from `followers`.  I could see `friends` being `SwoleMates` and `followers` being ... I don't know? haha!  Maybe just `Followers`?

Reply:
Update: Offline we talked about this.  Let's stick to the `followers` / `following` framework that is used on other social media platforms like Instagram and GitHub (for example).  No need to reinvent the wheel here.